helm/fluent-logshipping-app/values-cloudwatch.yaml

```diff
@@ -0,0 +1,22 @@
+# Needed to configure IRSA
+serviceAccount:
```

Comment:
Added an example

Reply:
Maybe it would be better to have it on a separate `sample` directory to match the other repos' architecture

Reply:
Sure

Reply:
Fixed :)